components/Home/HomeDownloadButton.tsx

```diff
@@ -15,8 +15,8 @@ const HomeDownloadButton = (props: HomeDownloadButtonProps) => {
   } = useNextraContext();
 
   const nodeDownloadLink = `https://nodejs.org/dist/${props.node}/`;
-  const nodeApiLink = `https://nodejs.org/dist/latest-${props.nodeMajor}/docs/api/`;
-
+  const nodeApiLink = `/latest-${props.nodeMajor}/docs/api/`;
+  const nodeAllDownloadsLink = `/download${props.isLts ? '/' : '/current'}`;
```

Comment:
Nice catch.

Reply:
While it was updated and tested by me, credit to @BethGriggs too for spotting where the error was in the original issue :-)